bin/ka-clone

```diff
@@ -39,15 +39,31 @@ def _cli_parser():
                         action='store_true',
                         help='attempt to khanify the current directory')
     # enable/disable functions
-    parser.add_argument('-p', '--protect-master',
-                        action='store_true',
-                        help='install hooks to protect the master branch')
-    parser.add_argument('--branch-name-hook',
-                        action='store_true',
-                        help='prepend commit-msgs with current branch name')
-    parser.add_argument('--lint-commit',
-                        action='store_true',
-                        help='hook up commit-msg linting')
+    group_protect_master = parser.add_mutually_exclusive_group()
+    group_protect_master.add_argument('-p', '--protect-master',
+                                      action='store_true',
+                                      help='install hooks to protect ' +
+                                           'the master branch')
+    group_protect_master.add_argument('--no-protect-master',
+                                      action='store_true',
+                                      help='do not install hooks to ' +
+                                           'protect the master branch')
```

Comment:
python3 added support for a cool new arg: `argparse.BooleanOptionalAction`.  See https://docs.python.org/3/library/argparse.html#argparse.BooleanOptionalAction. Then you don't need to define both of them.  The semantics are `--foo` sets it to true`, --no-foo` sets it to false, and otherwise it's None.

Reply:
@csilvers this was adding in 3.9, and it looks like we moved from 3.8 to 3.11 about a year ago (according to the set up scripts in dotfiles). Do you think everyone is on 3.9+ by now?

Reply:
Yes, we require everyone to be on 3.11 now, so it's safe to update.